ui/changes/src/index.js

```diff
@@ -233,6 +239,42 @@ function addRow(bugSummary) {
     risk_text.style.color = HIGH_RISK_COLOR;
     risk_text.textContent = 'Higher';
   }
+}
+
+async function populateVersions() {
+  var versionSelector = document.getElementById("releaseVersion")
+  versionSelector.addEventListener("change", () => {
+    setOption("releaseVersion", versionSelector.value);
+    rebuildTable();
+  });
+  let data = await landingsData;
+  var allVersions = new Set();
+    for (let date in data) {
+      for (let bug of data[date]) {
+        if (bug.versions.length) {
+          allVersions.add(...bug.versions)
+        }
+      }
+    }
+  var versions = [...allVersions]
+  versions.sort()
+  
+  function addOption(item, index, array) {
+    let opt = document.getElementById("releaseVersion")
+    let el = document.createElement("option")
+    if (item == array[array.length - 1]){
```

Comment:
If we switch to a multiple select, we can drop this code, along with all the LATEST_VERSION tracking as well since we can always just read the selected values off of the option. @marco-c does this sound right to you?

Reply:
Agreed!